### ISSUE TITLE
[BITV] 9.4.1.2/3.4 - The "File settings" button is an accordion button that should be implemented as such with the aria-expanded attribute. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37101]
See https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/ for further details. 

- [x] Fix
- [x] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v3-n4

</details>

- [ ] 9.4.1.2/10.4 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v10-n4

**Comment by susnux** [comment]
For NC27+ this is already fixed, as that button now opens a modal instead.
But still valid for NC25, did not check NC26, though.
